dataverse-persistence/src/main/resources/db/migration/V11__addedMissingTablesForTABFiles.sql

```diff
@@ -0,0 +1,73 @@
+create table variablemetadata
+(
+    id                   serial not null,
+    interviewinstruction text,
+    isweightvar          boolean,
+    label                text,
+    literalquestion      text,
+    notes                text,
+    universe             varchar(255),
+    weighted             boolean,
+    datavariable_id      bigint not null,
+    filemetadata_id      bigint not null,
+    weightvariable_id    bigint,
+    constraint variablemetadata_pkey
+        primary key (id),
+    constraint unq_variablemetadata_0
+        unique (datavariable_id, filemetadata_id),
+    constraint fk_variablemetadata_datavariable_id
+        foreign key (datavariable_id) references datavariable,
+    constraint fk_variablemetadata_filemetadata_id
+        foreign key (filemetadata_id) references filemetadata,
+    constraint fk_variablemetadata_weightvariable_id
+        foreign key (weightvariable_id) references datavariable
+);
+
+create index index_variablemetadata_datavariable_id
+    on variablemetadata (datavariable_id);
+
+create index index_variablemetadata_filemetadata_id
+    on variablemetadata (filemetadata_id);
+
+create index index_variablemetadata_datavariable_id_filemetadata_id
+    on variablemetadata (datavariable_id, filemetadata_id);
+
+create table categorymetadata
+(
+    id                  serial not null,
+    wfreq               double precision,
+    category_id         bigint not null,
+    variablemetadata_id bigint not null,
+    constraint categorymetadata_pkey
+        primary key (id),
+    constraint fk_categorymetadata_variablemetadata_id
+        foreign key (variablemetadata_id) references variablemetadata,
+    constraint fk_categorymetadata_category_id
+        foreign key (category_id) references variablecategory
+);
+
+create index index_categorymetadata_category_id
+    on categorymetadata (category_id);
+
+create index index_categorymetadata_variablemetadata_id
+    on categorymetadata (variablemetadata_id);
+
+create table vargroup
+(
+    id              serial not null,
+    label           text,
+    filemetadata_id bigint not null,
+    constraint vargroup_pkey
+        primary key (id),
+    constraint fk_vargroup_filemetadata_id
+        foreign key (filemetadata_id) references filemetadata
+);
+
+create index index_vargroup_filemetadata_id
+    on vargroup (filemetadata_id);
+
+ALTER TABLE datavariable
```

Comment:
Can we add this directly in datavariable table create statement (so we have it all in one place)?

Reply:
Hmm, in production environment it would be impossible, but since we are still in development it is different story. But I'm not sure if asking everyone to drop the database on all test environments is convenient (we also have 3 test environments and Krzysiek would have to clean 3 databases).

Reply:
That's true. I guess we could wrap ALTER statement in exist table check code and add columns only if they don't exist and then have correct, full CREATE statement.
But I'm not convinced it's worth the trouble. Will talk about it on daily since this change will possibly require changes in our test data scripts.